src/tx-builder.ts

```diff
@@ -258,10 +258,15 @@ export class TxBuilder {
         const tmpobj = tmp.dirSync({ unsafeCleanup: true });
```

Comment:
instead of using `tmp`, maybe just hardcode a tmp folder, and cleanup before reusing?

Reply:
yeah, should have no problem with it